quickstart-sda.md

```diff
@@ -0,0 +1,240 @@
+# Quickstart guide to the SDA
+
+This document contains information on how to work with the Sensitive Data Archive (SDA) from the GDI user perspective.
+
+## Brief description of the storage and interfaces stack
+
+The storage and interfaces software stack for the GDI-starter-kit consists of the following services:
+
+| Component     | Description |
+|---------------|------|
+| broker        | RabbitMQ based message broker, [SDA-MQ](https://github.com/neicnordic/sda-mq). |
+| database      | PostgreSQL database, [SDA-DB](https://github.com/neicnordic/sda-db). |
+| storage       | S3 object store, demo uses Minio S3. |
+| mock-oidc     | A python implementation of a mock-oidc in place of LS-AAI. |
```

Comment:
Should this really be here since it really should only exists in the `demo` profile

Reply:
Makes sense, `mock-oidc` is now removed from here.